manifests/noderesourcetopologymatch/node-crd.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: topology.node.k8s.io/v1alpha1
+kind: NodeResourceTopology
+metadata:
+  name: kind-worker
+topologyPolicy: "single-numa-node"
```

Comment:
I changed const name to CamelStyle, as I remember Tim Hockin suggested that.

Reply:
Ack, updated!